sktime/forecasting/adapters/_autots.py

```diff
@@ -0,0 +1,649 @@
+            },
+            {
+                "model_list": None,
+                "frequency": "infer",
+                "prediction_interval": 0.9,
+                "max_generations": 1,
+                "no_negatives": False,
+                "constraint": None,
+                "ensemble": "auto",
+                "initial_template": "General+Random",
+                "random_seed": 2022,
+                "holiday_country": "US",
+                "subset": None,
+                "aggfunc": "first",
+                "na_tolerance": 1,
+                "metric_weighting": {
+                    "smape_weighting": 5,
+                    "mae_weighting": 2,
+                    "rmse_weighting": 2,
+                    "made_weighting": 0.5,
+                    "mage_weighting": 0,
+                    "mle_weighting": 0,
+                    "imle_weighting": 0,
+                    "spl_weighting": 3,
+                    "containment_weighting": 0,
+                    "contour_weighting": 1,
+                    "runtime_weighting": 0.05,
+                    "oda_weighting": 0.001,
+                },
+                "drop_most_recent": 0,
+                "drop_data_older_than_periods": 100000,
+                "transformer_list": "auto",
+                "transformer_max_depth": 6,
+                "models_mode": "random",
+                "num_validations": "auto",
+                "models_to_validate": 1,
+                "max_per_model_class": None,
+                "validation_method": "seasonal 1",
+                "min_allowed_train_percent": 0.5,
+                "remove_leading_zeroes": False,
+                "prefill_na": None,
+                "introduce_na": None,
+                "preclean": None,
+                "model_interrupt": True,
+                "generation_timeout": None,
+                "current_model_file": None,
+                "verbose": -1,
+                "n_jobs": -2,
+                "model_name": "GLS",
+            },
+        ]
+        return params
+
+    def _instantiate_model(self):
+        """Instantiate the autoTS model.
+
+        Returns
+        -------
+            self: This function returns a reference to the instantiated model object.
+        """
+        if not isinstance(self.ensemble, list):
+            ensemble = [
+                self.ensemble,
+            ]
+        else:
+            ensemble = self.ensemble
+
+        if not isinstance(self.model_interrupt, tuple):
+            model_interrupt = (self.model_interrupt,)
+        else:
+            model_interrupt = self.model_interrupt
+
+        if self.model_name is not None and self.model_name != "":
+            logging.warning(
+                "The parameter model_name overwrites the model_list parameter."
+            )
+            model_list = [self.model_name]
+        else:
+            model_list = self.model_list
+
+        if self.metric_weighting is None or not self.metric_weighting != "":
```

Comment:
This is probably a typo, but why are you checking with `""`? Is it not supposed to be a dictionary?